frontend/components/Blocks/ParagraphBlock/ParagraphBlock.tsx

```diff
@@ -19,6 +20,10 @@ type Props = {
 };
 
 export default function Paragraph({ data }: Props) {
+  const router = useRouter();
+  //in wiki pages, width should be ignored
+  const isInWiki = router?.query?.path?.includes("wiki");
```

Comment:
Dit is een beetje een dubieuze oplossing aangezien het dus stuk gaat als je wikipagina's niet meer onder /wiki/ zitten. Het is logischer om te kijken naar het paginatype, ik weet alleen niet of deze beschikbaar is in het component.